Spawn-App-iOS-SwiftUI/ViewModels/ChoosingTags/ChoosingTagViewModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ChoosingTagViewModel.swift
+//  Spawn-App-iOS-SwiftUI
+//
+//  Created by Michael Tham on 23/1/25.
+//
+
+import Foundation
+
+class ChooseTagPopUpViewModel: ObservableObject {
+	@Published var chooseTagErrorMessage: String = ""
+    @Published var tags: [UUID] = []
+    @Published var selectedTags: Set<UUID> = []
+    
+    var userId: UUID
+
+    var apiService: IAPIService = MockAPIService()
```

Comment:
```suggestion
    var apiService: IAPIService
```
Where using dependency injection through the constructor; no need for this